### PR TITLE
Fix answers not changing value from 'correct' to anything else.

### PR DESCRIPTION
**Pull request by FrenjaminBanklin**
Closes https://github.com/ucfopen/Materia/issues/1187.

Correctly unsets 'options.correct' on answers when value is set below 100.

Also cleaned up some tests that were failing due to prior changes.

### CODE REVIEW

File: src/controllers/creator.coffee [hunk 1]
```diff
@@ -393,6 +397,7 @@ Enigma.controller 'enigmaCreatorCtrl', ['$scope', '$timeout', ($scope, $timeout)
 		problems = []
 
 		for answer in question.answers
+			console.log answer.options.correct, answer.value
```
Comment (clpetersonucf) [inline src/controllers/creator.coffee]: left a console.log in here

**Review by clpetersonucf — CHANGES_REQUESTED** [review]
LGTM once that extra console.log is snipped

**Review by clpetersonucf — APPROVED** [review]

**Comment by clpetersonucf** [comment]
:+1:

**Review by iturgeon — APPROVED** [review]
couldn't cause any issues here, and we need them tests

**Pull request merged**